sdk/storage/azblob/internal/exported/hashing_polynomial.go

```diff
@@ -0,0 +1,15 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package exported
+
+import (
+	"hash/crc64"
+)
+
+const CRC64Polynomial uint64 = 0x9A6C9329AC4BC9B5
```

Comment:
Looks like this, and `CRC64Table`, don't actually need to be exported from this package.

Also, it might be good to condense all of this content into one file as it's all related.

Reply:
I am going to condense it to one file, but I intend to leave them exported, as they are used in documentation for the package.